docs/File-Formats.md

```diff
@@ -126,10 +126,10 @@ The first four rows of the clinical data file contain tab-delimited metadata abo
 - Row 3: **The attribute Datatype**: The datatype of each clinical attribute (must be one of:  STRING, NUMBER, BOOLEAN)
 - Row 4: **The attribute Priority**: A number which indicates the importance of each attribute.  In the future, higher priority attributes will appear in more prominent places than lower priority ones on relevant pages (such as the [Study View](http://www.cbioportal.org/study?id=brca_tcga)). A lower number indicates a higher priority.
     ```
-    To promote certain chart in study view, please increase priority in the database to certain number. The higher the score, the higher priority it will be displayed in the study view.
+    To promote certain chart in study view, please increase priority to certain number. The higher the score, the higher priority it will be displayed in the study view.
     If you want to hide chart, please set the priority to 0. For combination chart, as long as one of the clinical attribute has been set to 0, it will be hidden.
     
-    Currently, we preassigned priority to few charts, but as long as you assign a priority in the database except than 1, these preassigned priorities will be overwritten.
+    Currently, we preassigned priority to few charts, but as long as you assign a priority except than 1, these preassigned priorities will be overwritten.
```

Comment:
You are correct. Because the current setting is using 1 as default, so we decided to use 1 as default in the front-end as well. Any integer would work, only 1 and 0 are treated specially